network_manager/node_registry.py

```diff
@@ -3,18 +3,17 @@
 
 
 class NodeRegistry:
-
     # Gets node_registry.json from github and returns the content
-    def get_node_registry(self):
-
+    @staticmethod
```

Comment:
It's not a bad thing to put static methods in classes but they should be decorated as such.